### PR TITLE
MVP CGM in trends view

### PR DESCRIPTION
**Pull request by jebeck**
Basically just two sets of changes here:
- smallish styling and rendering changes for the smbg version of trends/modal view to match the cbg versions in @tidepool/viz
- small preprocessing changes in the TidelineData constructor to add the `msPer24` property to cbgs as well as smbgs

Dependency of [blip #320](https://github.com/tidepool-org/blip/pull/320).

### CODE REVIEW

**Comment by jebeck** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;qQqZiU3z&#x2F;3-ux-2-dev-8-tst-3-blip-cgm-in-trends-mvp-release-candidate"> &lt;ux:2 dev:8 tst:3&gt; [blip] [cgm in trends] MVP release candidate</a>

**Comment by jebeck** [comment]
This is ready for review now @jh-bate and @krystophv 

Dependency of [blip #320](https://github.com/tidepool-org/blip/pull/320). Probably best to sign off here but wait to let me merge when ready to publish to npm and merge on blip.

File: js/tidelinedata.js [hunk 1]
```diff
@@ -20,6 +20,7 @@
 var _ = require('lodash');
 var crossfilter = require('crossfilter');
 var d3 = require('d3');
+var moment = require('moment-timezone');
```
Comment (krystophv) [inline js/tidelinedata.js]: It looks like this is never used?

**Comment by krystophv** [comment]
Yeah, this LGTM 👍

**Pull request merged**